docs/proposals/2019-02-14_Dropcap.md

```diff
@@ -0,0 +1,44 @@
+# Dropcap for paragraph and heading elements
+
+# Problem
+
+Users want the ability to add a dropcap to paragraphs and headings.
+
+# Proposal
+
+we propose adding a new trait, `format` to text and heading content elements, which will contain the new property `dropcap`.
```

Comment:
Did you mean `header`?

https://github.com/washingtonpost/ans-schema/blob/master/src/main/resources/schema/ans/0.9.1/story_elements/header.json

Reply:
Yes